app/Console/Commands/FlightData.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Models\Flight;
+
+class FlightData extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'flight:update';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'This will update the flight information';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $queryString = http_build_query([
+            'api_key' => config('services.key.api'),
+            'arr_iata' => 'KBP'
```

Comment:
should be passed as the command parameter with default value, instead of hardcoding